<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dicomTags</class>
 <widget class="QMainWindow" name="dicomTags">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ctkExpandableWidget" name="ResizableFrame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>841</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>791</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="ctkPathListWidget" name="PathListWidget">
      <property name="paths">
       <stringlist>
        <string>C:/</string>
       </stringlist>
      </property>
     </widget>
     <widget class="QTableView" name="tableView"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionApply_Change"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionApply_Change">
   <property name="text">
    <string>Apply Change</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkExpandableWidget</class>
   <extends>QFrame</extends>
   <header>ctkExpandableWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathListWidget</class>
   <extends>QListView</extends>
   <header>ctkPathListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
